src/fgimic/jfield.f90

```diff
@@ -425,19 +421,29 @@ subroutine jvector_plots(this,tag)
             ! read(GRIDELE, '(3i4)') ncells, dummy, dummy
             read(GRIDELE, *) ncells, dummy, dummy
             ! write(*,*) "ncells ", ncells
-            allocate(cells(4,ncells))
+            allocate( this%grid%cells(4,ncells))
```

Comment:
Here the inclusion of the cells makes sense. On the other hand we only need this when reading in grid.1.ele.....